test/t_web/channels/feed_channel_test.exs

```diff
@@ -1559,6 +1557,7 @@ defmodule TWeb.FeedChannelTest do
                  "id" => m1.id,
                  "profile" => %{name: "mate-1", story: [], user_id: p1.id, gender: "F"},
                  "audio_only" => false,
+                 "expiration_date" => ~U[2021-10-07 12:16:05Z],
```

Comment:
this test doesn't make much sense, but will be removed in #546

Reply:
please add a short comment about why it doesn't make sense on top of the test and reference it in the issue

Reply:
thank you

Reply:
done